streams/src/main/java/org/apache/kafka/streams/kstream/Transformer.java

```diff
@@ -82,6 +84,7 @@
     R transform(final K key, final V value);
 
     /**
+     * @deprecated As of 0.11.1.0 please use {@link Punctuator} functional interface instead.
```

Comment:
Nit: remove `As of 0.11.1.0` -- we also don't know yet what the next version number will be. If it's not 0.11.1.0 it would be wrong and I am pretty sure we will miss to update it.